src/main/predict.cpp

```diff
@@ -204,8 +204,22 @@ void go(Config config) {
     }
     if(verbose) cout << "inputFile: '" << config.inputFile << "'"<< endl;
     if(config.inputFile == "") {
-        cin.read(reinterpret_cast< char * >(inputData), inputCubeSize * config.batchSize * 4l);
-        more = !cin.eof();
+	const int size = inputCubeSize * config.batchSize * 4l;
+	char *bytes = new char[size];
+
+	cin.read(bytes, size);
```

Comment:
Hmmm, isnt this the same as before?  I thought you were looping over this or something, but it seems like its the exact same commandas before?